versioned_docs/version-1.3.2/docs/rules/engines/ast-engine.md

```
@@ -0,0 +1,350 @@
+---
+sidebar_position: 2
+---
+
+# AST Engine
+
+The AST engine is responsible for parsing the steps of a rule with type `ast`. Following is an example of what the headers of a rule would look like:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)